util/Order.cpp

```diff
@@ -804,6 +804,103 @@ bool BombardOrder::UndoImpl() const {
 }
 
 ////////////////////////////////////////////////
+// DestroyPlanetOrder
+////////////////////////////////////////////////
+DestroyPlanetOrder::DestroyPlanetOrder() :
```

Comment:
why is this needed, as opposed to making it a bombard effect...?

Reply:
I don't understand. Can you elaborate how that might work?

Reply:
Make the bombard effect of a part destroy a planet. When told to bombard a planet, that ship will destroy it.

Reply:
There might be cases where a ship is capable of destroying as well as bombarding a planet.

Reply:
That can be relevant for a variety of bombard effects that a player .ay notalways eant to use all together or not at all. The solution should probably be more general and adaptable than this whixh is specific to a particular part.

Reply:
Still not sure if I completely understand what you mean. Do you mean I should merge the DestroyPlanet Order into the BombardOrder, e. g. by adding a bool is_destroy_order, which is checked to determine if it is actually a bopmbard or a planet destruction order?

Reply:
Not looked into the idea fully, could possibly have a base order specialized by script definitions.  That concept may not suitable for the scope of this PR though.

Reply:
If it's important to be able to use only particular bombard weapons when bombarding, then there should be a general way to do this. Your particular new bombard weapon probably doesn't warrant its own separate order class, when there could be various other similar bombard weapons as well.